src/status_im/utils/datetime.cljs

```diff
@@ -10,6 +10,8 @@
             [clojure.string :as s]
             [status-im.goog.i18n :as goog.18n]))
 
+;;;; Datetime constants
```

Comment:
nit: are these really constants? 🤔

Reply:
You're right, most of them are constants (time units), but others are not. I would gladly take a suggestion here.